.travis.yml

```diff
@@ -1,31 +1,46 @@
 language: minimal
+sudo: required
+
 if: branch = master
-before_install:
-- curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
-- chmod 700 get_helm.sh
-- "./get_helm.sh"
-script:
-- mkdir releases
-- cd charts
-- for i in $(find . -maxdepth 1 -mindepth 1 -type d) ; do helm lint $i && helm dependency update $i 
-  && helm package $i -d ../releases ; done
-- cd ..
-- git checkout gh-pages
-- mv *.tgz releases/
-- cd releases
-- helm repo index --url https://mozilla-it.github.io/helm-charts/ --merge ../index.yaml .
-deploy:
-  provider: pages
-  github_token: $GITHUB_TOKEN
-  local_dir: releases
-  verbose: true
-  keep_history: true
-  skip_cleanup: true
-  target_branch: gh-pages
-  on:
-    branch: master
+
+stages:
+  - Lint and Validate
+  - Deploy
+
+before_script:
+  - ./ci/setup.sh
+
+jobs:
+  - stage: Lint and Validate
+    script:
+      - ./ci/lint.sh
+  # TODO: Move this over to a script
+  - stage: Deploy
+    if: type = push AND branch = master
+    script:
```

Comment:
You already have committed the script running all the commands below, why don't just use it?

Reply:
Fixed in f30fc77f920c673d3b0bd32ce911be8d9efa25ca